build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
```

Comment:
All of our public build scripts should be Kotlin - https://gradle.slack.com/archives/C03A0MT6S/p1575851014215200

Reply:
Initial work with 1d54211e41831acda1834bf4429b36d81398fd85, converted sample build script in a later commit.